### PR TITLE
Update dap collector

### PR DESCRIPTION
**Pull request by simon-friedberger**
Matches newer Janus version and is parallelized

Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is
  referenced, the pull request should include the bug number in the title)
- [ ] Scan the PR and verify that no changes (particularly to
  `.circleci/config.yml`) will cause environment variables (particularly
  credentials) to be exposed in test logs
- [ ] Ensure the container image will be using permissions granted to
  [telemetry-airflow](https://github.com/mozilla/telemetry-airflow/)
  responsibly.

### CODE REVIEW

**Comment by simon-friedberger** [comment]
Also tracked as part of https://mozilla-hub.atlassian.net/browse/FXCTO-13

File: jobs/dap-collector/dap_collector/main.py [hunk 1]
```diff
@@ -1,124 +1,174 @@
...
+import requests
+
+LEADER = "https://dap-07-1.api.divviup.org"
+CMD = f"./collect --task-id {{task_id}} --leader {LEADER} --vdaf {{vdaf}} {{vdaf_args}} --authorization-bearer-token {{auth_token}} --batch-interval-start {{timestamp}} --batch-interval-duration {{duration}} --hpke-config {{hpke_config}} --hpke-private-key {{hpke_private_key}}"
+INTERVAL_LENGTH = 300
+
+
+def read_tasks():
+    """Read task configuration from Google Cloud bucket."""
+    url = "https://storage.googleapis.com/collector-config-prod/tasks.json"
```
Comment (akkomar) [inline jobs/dap-collector/dap_collector/main.py:22]: It's fine for the PoC but should be configurable via a parameter.

**Review by akkomar — APPROVED** [review]
r+wc

File: jobs/dap-collector/dap_collector/main.py (hunk 1 above)
Comment (simon-friedberger) [reply jobs/dap-collector/dap_collector/main.py]: Done!

**Pull request merged**